### PR TITLE
added the new firebase performance

### PR DESCRIPTION
**Pull request by ahmedHusseinF**

### CODE REVIEW

File: plugin.js [hunk 1]
```diff
@@ -48,4 +49,9 @@ export default (ctx, inject) => {
     inject('fireMess', fireMess)
   }
 
+  if(process.browser && options.useOnly.includes('performance')){
+    const perf = firebase.performance()
```
Comment (lupas) [inline plugin.js]: rename to firePerf

const firePerf = firebase.performance()

File: plugin.js [hunk 2]
```diff
@@ -48,4 +49,9 @@ export default (ctx, inject) => {
...
+    inject('perf', perf)
```
Comment (lupas) [inline plugin.js]: rename to firePerf

inject('firePerf', firePerf)

**Comment by ahmedHusseinF** [comment]
done 👍 and waiting for another round of review

File: README.md [hunk 3]
```diff
@@ -25,7 +26,7 @@ Make sure you have Nuxt and Firebase installed in your project.
 ```json
 "dependencies": {
   "nuxt": "^2.6.2",
-  "firebase": "^5.10.0"
+  "firebase": "^6.0.2"
```
Comment (lupas) [inline README.md]: Thanks for that :)

**Comment by lupas** [comment]
@ahmedHusseinF 
Thanks a lot! :) I merged it into the "addFirePerf" branch since I want to do the following modifications before merging it into master:

1) Remove semi-colons and fixed single-quote that you (I assume unintentionally) added.
(not your fault, I should add a lint/prettier config)

2) Since `appId` is now new in the requiredKeys array, this is a breaking change. People will have to add that to their config. So I'll check if there is a way around that o

3) Add the object itself as this.$firePerfObj, as we do for all the other services.

Thanks again, will publish very shortly,

**Pull request merged**